pipeline_steps/tempest.groovy

```diff
@@ -1,25 +1,19 @@
-def tempest_install(vm=null){
+def tempest_install(){
   // NOTE(mkam): Can remove ANSIBLE_CACHE_PLUGIN when we no longer gate stable/mitaka
   // NOTE(alextricity25): --skip-tags needs to be used here because tags cannot be used
   // with included roles in earlier versions of Ansible. Such that used in Mitaka
   common.openstack_ansible(
-    vm: vm,
     playbook: "../../scripts/run_tempest.yml",
     args: "--skip-tags tempest_execute_tests",
     path: "/opt/rpc-openstack/rpcd/playbooks",
     environment_vars: ["ANSIBLE_CACHE_PLUGIN=memory"]
   )
 }
 
-def tempest_run(wrapper="") {
+def tempest_run() {
   tempest_cmd = "cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible ../../scripts/run_tempest.yml -t tempest_execute_tests -vv"
-  if (wrapper != "") {
-    tempest_cmd_wrapped = "${wrapper} '${tempest_cmd}'"
-  } else {
-    tempest_cmd_wrapped = tempest_cmd
-  }
   def output = sh (script: """#!/bin/bash
-  ${tempest_cmd_wrapped}
+  ${tempest_cmd}
```

Comment:
There's no need for this tempest_cmd variable any more, it can go straight in this script. Then it doesn't have to be on one line.